pythonAPIClient/client.py

```diff
@@ -34,7 +34,7 @@ class Client(object):
         at https://www.cam2project.net/.
 
         For each methods except internal method like _check_token(),
-        those methods will rerun request_token() to get a new token if token expires.
+        those methods will rerun _request_token() to get a new token if token expires.
```

Comment:
Did we rename to _request_token() to follow a convention? What was the reasoning?

Reply:
that is a python way to define a private class method, signaling that user should not use this method. request token is used by other class methods, and we do not expect user to call it. (although there is no mechanism in python to prevent user from doing that. ) At least by doing that we can remove this function from documentation; sphinx by default omit all __special_methods__ and _private_methods.